### PR TITLE
RPG-7V2 Field Manual

### PR DESCRIPTION
**Pull request by beaar**
TBG-7V button not working

### CODE REVIEW

File: addons/tm_bear_fieldmanual_rpg7v2/CfgVehicles.hpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	class Man;
+	class CAManBase: Man 
+	{
+		class ACE_SelfActions
+		{
+			class ADDON
+			{
+				displayName = "RPG-7V2 Field Manual";
+				condition = "(secondaryWeapon player) isEqualTo 'rhs_weap_rpg7'";
+				statement = "0 = 0 spawn {findDisplay 46 createDisplay 'RscBear_fieldManual_rpg7v2';};";
```
Comment (headswe) [inline addons/tm_bear_fieldmanual_rpg7v2/CfgVehicles.hpp]: I dont think you need ``0 = 0 spawn..``  but simpley ``findDisplay 46 createDisplay 'RscBear_fieldManual_rpg7v2';``

**Comment by beaar** [comment]
Everything should be working now. :moneybag: :moneybag: :moneybag:

File: addons/tm_bear_fieldmanual_rpg7v2/CfgVehicles.hpp [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	class Man;
+	class CAManBase: Man 
+	{
+		class ACE_SelfActions
+		{
+			class bear_fieldmanual_rpg7v2
+			{
+				displayName = "RPG-7V2 Field Manual";
+				condition = "(secondaryWeapon player) isEqualTo 'rhs_weap_rpg7'";
+				statement = "0 = 0 spawn {findDisplay 46 createDisplay 'RscBear_fieldManual_rpg7v2';};";
```
Comment (headswe) [inline addons/tm_bear_fieldmanual_rpg7v2/CfgVehicles.hpp]: This is still wonky looking, should be doable to trim down to ``spawn {findDisplay 46 createDisplay 'RscBear_fieldManual_rpg7v2';};``

File: addons/tm_bear_fieldmanual_rpg7v2/functions/fn_select.sqf [hunk 3]
```diff
@@ -0,0 +1,17 @@
+disableSerialization;
+params ["_magazine"];
+(missionNamespace getVariable format ["bear_fieldManual_rpg7v2_%1", _magazine]) params [["_idc", 834111], ["_reticle", "l"]];
```
Comment (headswe) [inline addons/tm_bear_fieldmanual_rpg7v2/functions/fn_select.sqf]: Just use a switch instead??

```
_data = switch (_magazine) do {
    case "rocket0": { [...]  };
    case "rocket1": { [..] };
};
_data params [["_idc", 834111], ["_reticle", "l"]];
```

File: addons/tm_bear_fieldmanual_rpg7v2/functions/fn_init.sqf [hunk 4]
```diff
@@ -0,0 +1,6 @@
+bear_fieldManual_rpg7v2_rhs_rpg7_PG7V_mag = [834111, "l"];
```
Comment (headswe) [inline addons/tm_bear_fieldmanual_rpg7v2/functions/fn_init.sqf]: Pointless!

**Pull request merged**